src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/SQLFunctionsIT.java

```diff
@@ -159,6 +160,44 @@ public void caseChangeWithAggregationTest() throws IOException {
                 hitAny("/aggregations/UPPER_1/buckets", kvString("/key", equalTo("AMBER"))));
     }
 
+    @Test
+    public void castIntFieldToDoubleWithoutAliasTest() throws IOException {
+        String query = "SELECT CAST(age AS DOUBLE) FROM " + TestsConstants.TEST_INDEX_ACCOUNT + " /account limit 1";
+
+        final SearchHit[] hits = query(query).getHits();
+        assertTrue(hits[0].getFields().containsKey("cast_age"));
+        assertTrue(hits[0].getFields().get("cast_age").getValue() instanceof Double);
+    }
+
+    @Test
+    public void castIntFieldToDoubleWithAliasTest() throws IOException {
+        String query = "SELECT CAST(age AS DOUBLE) AS test_alias " +
+                       "FROM " + TestsConstants.TEST_INDEX_ACCOUNT + " /account limit 1";
+
+        final SearchHit[] hits = query(query).getHits();
+        assertTrue(hits[0].getFields().containsKey("test_alias"));
+        assertTrue(hits[0].getFields().get("test_alias").getValue() instanceof Double);
+    }
+
+    @Test
+    public void castIntFieldToStringWithoutAliasTest() throws IOException {
+        String query = "SELECT CAST(balance AS STRING) FROM " + TestsConstants.TEST_INDEX_ACCOUNT + " /account limit 1";
+
+        final SearchHit[] hits = query(query).getHits();
+        assertTrue(hits[0].getFields().containsKey("cast_balance"));
+        assertTrue(hits[0].getFields().get("cast_balance").getValue() instanceof String);
+    }
+
+    @Test
+    public void castIntFieldToStringWithAliasTest() throws IOException {
```

Comment:
Append `FROM` into first select string. Here and other new test cases. Similar to `castIntFieldToStringWithoutAliasTest ` above.

Reply:
Then I believe checkstyle should fail for `castIntFieldToStringWithoutAliasTest()` as well.

Reply:
that String statement is actually at the maximum allowed length but any more and it would fail

Reply:
Sorry if it was not clear, I meant making it `SELECT CAST(balance AS STRING) AS cast_string_alias FROM ` instead of `"SELECT CAST(balance AS STRING) AS cast_string_alias"  + "FROM"` and keep the `TestsConstants.TEST_INDEX_ACCOUNT + " /account limit 1"` on next line. This way it would be consistent with other new tests added without failing checkstyle.

PS: It's a minor change , you can skip if you like.